packages/xo-web/src/common/xo/index.js

```diff
@@ -628,6 +630,26 @@ export const subscribeCloudXoConfig = createSubscription(() =>
   fetch('./rest/v0/cloud/xo-config').then(resp => resp.json())
 )
 
+const subscribeSrsXostorInterfaces = {}
+export const subscribeXostorInterfaces = sr => {
+  const srId = resolveId(sr)
+
+  if (subscribeSrsXostorInterfaces[srId] === undefined) {
+    subscribeSrsXostorInterfaces[srId] = createSubscription(() => _call('xostor.getInterfaces', { sr: srId }))
+  }
+
+  return subscribeSrsXostorInterfaces[srId]
+}
+subscribeXostorInterfaces.forceRefresh = sr => {
+  if (sr === undefined) {
+    forEach(subscribeSrsXostorInterfaces, subscription => subscription.forceRefresh())
+    return
+  }
+
+  const subscription = subscribeSrsXostorInterfaces[resolveId(sr)]
+  subscription?.forceRefresh()
```

Comment:
```suggestion
  subscription?.forceRefresh?.()
```

But actually: should we create the subscription if it doesn't exist?

Reply:
Why would `forceRefresh` be `undefined`?
Based on what we usually do for the `forceRefresh` function of a subscription, if the `subscription` doesn't exist, we don't create it.